src/markdown/schema/schema-caching.md

```diff
@@ -0,0 +1,111 @@
+# Schema Caching
+
+## Schema Caching Behaviour
+
+When a connection is made to an Amazon DocumentDB database, the Amazon DocumentDB JDBC driver 
+checks for a previously cached version of the mapped schema. If a previous version exists, 
+the latest version of the cached schema is read and used for all further interaction with the database.
+
+If a previously cached version does not exist, the process of [schema discover](schema-discovery.md) is automatically
+started on all the accessible collections in the database. The discovery process uses the properties 
+`scanMethod` (default `random`), and `scanLimit` (default `1000`) when sampling documents from the database.
+At the end of the discovery process, the resulting schema mapping is written to the cache using the name
+associated with the property `schemaName` (default `_default`).

```

Comment:
Add link to instructions on how the user can configure those properties to a non-default value?
E.g., "For how to configure `scanMethod`, `scanLimit`, and `schemaName`, check [this page](url) to learn more about the values"